public/components/datasources/components/manage/associated_objects/associated_objects_tab.tsx

```diff
@@ -325,24 +336,28 @@ export const AssociatedObjectsTab: React.FC<AssociatedObjectsTabProps> = (props)
         columns: table.columns,
       };
     });
-    const accelerationObjects: AssociatedObject[] = cachedAccelerations
-      .filter((acceleration: CachedAcceleration) => acceleration.database === selectedDatabase)
-      .map((acceleration: CachedAcceleration) => ({
-        tableName: acceleration.table,
-        datasource: datasource.name,
-        id: acceleration.indexName,
-        name: getAccelerationName(acceleration),
-        database: acceleration.database,
-        type: ACCELERATION_INDEX_TYPES.find((accelType) => accelType.value === acceleration.type)!
-          .value as AssociatedObjectIndexType,
-        accelerations:
-          acceleration.type === 'covering' || acceleration.type === 'skipping'
-            ? tableObjects.find(
-                (tableObject: AssociatedObject) => tableObject.name === acceleration.table
-              )
-            : [],
-        columns: undefined,
-      }));
+    // For data connections using lake formation we don't want to show accelerations, so we simply assign empty array
+    const accelerationObjects: AssociatedObject[] = isS3ConnectionWithLakeFormation
```

Comment:
so it seems like the current s3 datasource connection will still have the original design of both tables and accelerations for the associated objects tab?